packages/es-components/src/components/navigation/Anchor.js

```diff
@@ -13,8 +13,7 @@ const StyledAnchor = styled.a`
   }
 `;
 
-function Anchor({ ...props }) {
-  return <StyledAnchor {...props} />;
-}
-
+const Anchor = React.forwardRef((props, ref) => (
+  <StyledAnchor {...props} ref={ref} />
+));
```

Comment:
I know this is literally just a pass-through, but styleguidist gets mad if I don't do something like this.

Reply:
Would [what we did](https://github.com/WTW-IM/es-components/blob/a38c98262111b29edfa9b3963256b552f6540b10/packages/es-components/src/components/patterns/screenReaderOnly/screenReaderOnly.js#L17) for `screenReaderOnly` work here?

Reply:
I can try that!

Reply:
It worked, but I had to make sure the export was _named_ Anchor. Otherwise styleguidist would show `StyledAnchor` instead, which would be confusing.